bitvm/src/signatures/signing_winternitz.rs

```diff
@@ -21,6 +20,18 @@ pub struct WinternitzSecret {
 /// Bits per digit.
 pub const LOG_D: u32 = 4;
 
+/// Winternitz verifier, returns the message in digits
+pub const WINTERNITZ_MESSAGE_VERIFIER: Winternitz<ListpickVerifier, VoidConverter> =
+    Winternitz::new();
+
+/// Winternitz verifier, returns the message in bytes
+pub const WINTERNITZ_VARIABLE_VERIFIER: Winternitz<ListpickVerifier, ToBytesConverter> =
+    Winternitz::new();
+
+/// Winternitz verifier for compact signature representation, returns the message in bytes
+pub const WINTERNITZ_MESSAGE_COMPACT_VERIFIER: Winternitz<BruteforceVerifier, VoidConverter> =
+    Winternitz::new();
+
```

Comment:
For the sake of consistency with other docstrings in this module:
```suggestion
/// Winternitz verifier, returns the message in digits.
pub const WINTERNITZ_MESSAGE_VERIFIER: Winternitz<ListpickVerifier, VoidConverter> =
    Winternitz::new();

/// Winternitz verifier, returns the message in bytes.
pub const WINTERNITZ_VARIABLE_VERIFIER: Winternitz<ListpickVerifier, ToBytesConverter> =
    Winternitz::new();

/// Winternitz verifier for compact signature representation, returns the message in bytes.
pub const WINTERNITZ_MESSAGE_COMPACT_VERIFIER: Winternitz<BruteforceVerifier, VoidConverter> =
    Winternitz::new();

```